dist/tester.html

```diff
@@ -1,23 +0,0 @@
-<html>
```

Comment:
Why was this file deleted?  It's not that it's super important but it did serve a purpose in that one could load this in a browser and verify the built browser binaries worked.

Reply:
This is an oversight, I'll restore it